packages/compiler/test/server/rename.test.ts

```diff
@@ -13,23 +13,23 @@ describe("compiler: server: rename and find references", () => {
   test(
     "models",
     `
-      model Model/**/ {
-        prop: Mo┆del/**/; 
+      model MyModel/**/ {
```

Comment:
found a bug with the find references which cause the `Model` reflection type to be discovered here Filled an issuer here https://github.com/microsoft/cadl/issues/1162